code/modules/unit_tests/create_and_destroy.dm

```diff
@@ -38,6 +38,7 @@ GLOBAL_VAR_INIT(running_create_and_destroy, FALSE)
 		///this instant starts a timer, and if its being instantly deleted it can cause issues
 		/obj/machinery/atm,
 		/datum/hotspot,
+		/obj/machinery/ocean_elevator,
```

Comment:
this seems unrelated

Reply:
it is, but i needed to change it for linters so i put it here im the king of unrelated changes lol